File: scala-play-server/app/model/ResponseBox.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ResponseBox.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseBox(
  minLat: Double,
  maxLat: Double,
  minLng: Double,
  maxLng: Double
)

object ResponseBox {
  implicit lazy val responseBoxJsonFormat: Format[ResponseBox] = Json.format[ResponseBox]
}

File: fsharp-functions/OpenAPI/src/model/ResponseBox.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json

module ResponseBox = 

  //#region ResponseBox

  [<CLIMutable>]
  type ResponseBox = {
    [<JsonProperty(PropertyName = "min_lat")>]
    MinLat : double;
    [<JsonProperty(PropertyName = "max_lat")>]
    MaxLat : double;
    [<JsonProperty(PropertyName = "min_lng")>]
    MinLng : double;
    [<JsonProperty(PropertyName = "max_lng")>]
    MaxLng : double;
  }
  
  //#endregion
  